docs/en/query_language/functions/bitmap_functions.md

```diff
@@ -56,26 +56,38 @@ SELECT bitmapToArray(bitmapBuild([1, 2, 3, 4, 5])) AS res
 └─────────────┘
 ```
 
-## bitmapSubsetInRange {#bitmap_functions-bitmapsubsetinrange}
+## bitmapSubsetInRange {#bitmapsubsetinrange}
 
-Return subset in specified range (not include the range_end).
+Converts a bitmap into an integer array. Returns values from given range not including the range_end.
```

Comment:
* range_end is referenced before it was introduced in example
* "Converts a bitmap into an integer array" sounds misleading, following explanation demonstrates different behaviour